src/Test/entities_testing/JournalTest.java

```diff
@@ -0,0 +1,67 @@
+import entities.RecipeEntry;
+import entities.Journal;
+import entities.Entry;
+import entities.Recipe;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import junit.framework.TestCase;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static java.util.Arrays.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JournalTest extends TestCase {
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        super.setUp();
+    }
+
+    @AfterEach
+    public void tearDown() throws Exception {
+        super.tearDown();
+    }
+
+    @Test
+    void sortEntriesChronologically1() {
+        RecipeEntry entry1 = new RecipeEntry(Instant.now(),null);
+        RecipeEntry entry2 = new RecipeEntry(Instant.now(), null);
+        RecipeEntry entry3 = new RecipeEntry(Instant.now(), null);
+        RecipeEntry entry4 = new RecipeEntry(Instant.now(), null);
+        RecipeEntry[] array1 = {entry3, entry2, entry1, entry4};
+        RecipeEntry[] array_expected = {entry1, entry2, entry3, entry4};
+        Journal journal = new Journal(array1);
+        journal.sortEntriesChronologically();
+        assertEquals(array1, array_expected);
```

Comment:
Is this testing that the array is sorted or that the journal is sorted? Am a bit confused about this.